application/controllers/handler.js

```diff
@@ -345,15 +352,27 @@ module.exports = {
   },
 
   getPublicUser: (req, res) => {
-    let identifier = decodeURI(req.params.identifier);
+    let identifier = decodeURI(req.params.identifier).replace(/\s/g,'');
```

Comment:
As the identifier is an integer or a string I validate with the correct regex if integer validations fails